doc/source/operations/secret-rotation.rst

```diff
@@ -370,10 +370,12 @@ Full method
 
       kayobe overcloud service deploy
 
-20. Manually update ``heat_domain_admin_password``
+20. Manually update ``heat_domain_admin_password``, using the newly generated
+    OpenStack Admin credentials.
 
-    #. TODO: Instructions
-       This has not been tested yet
+    .. code:: bash
+
+       openstack user set --password <password> heat_domain_admin
```

Comment:
Do you not need to set the domain when doing this?

```suggestion
       openstack user set --domain heat_user_domain --password <password> heat_domain_admin
```

Reply:
Done